roles/infrastructure/tasks/hub.yml

```diff
@@ -0,0 +1,68 @@
+---
+- name: Get existing VMs
+  block:
+    - name: Include get vms task
+      ansible.builtin.include_tasks: vms/get.yml
+      vars:
+        app_tag: hub
+        private_host_groups:
+          - aap
+          - hub_private
+        public_host_groups:
+          - hub_public
+      tags:
+        - hub
+
+    - name: Add public IP host to installer
+      when:
+        - infrastructure_running_virtual_machine_public_ip is defined
+        - infrastructure_running_virtual_machine_public_ip.results | length > 0
+        - (groups.installer is not defined or groups.installer | length == 0)
+      ansible.builtin.add_host:
+        groups:
+          - installer
+        name: "{{ infrastructure_running_virtual_machine_public_ip.results[0].publicipaddresses[0].ip_address }}"
+      tags:
+        - hub
+
+- name: Set number of Hub VMs to create
```

Comment:
According to the OPL, hub and controller should not be capitalized in this case. https://docs.google.com/spreadsheets/d/1DLS_lS3VKidgZIvcLmLp9BoiqptkvqHWfe1D5FD2kfk/edit#gid=1375785039&range=A167
```suggestion
- name: Set number of hub VMs to create
```